README.md

```diff
@@ -54,6 +54,6 @@ This will require multiple terminal windows.  A screen multiplexer such as [tmux
 
 * Ensure your DigiByte node is running.  It is recommended that you do not specify an rpcpassword in digibyte.conf.  The rpcuser and rpcpassword options will soon be deprecated.
 * In one terminal, go to the ``src`` directory and run ``./autocompile.sh --testnet cyclone_v_gx_starter_kit de10_nano``
-* In another terminal, go to the ``src/pool/solo`` directory and run ``python pool.py --testnet <dgb_address>``
+* In another terminal, go to the ``src/pool/solo`` directory and run ``python pool.py -st --testnet <dgb_address>``
```

Comment:
I don't think the common user will want this flag.  It can fill up the screen rather quickly and push any important error messages off the top.